CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+
+### Updated
```

Comment:
We haven't done a good job of this in the past, but let's not use the term `Updated`. [The guide](http://keepachangelog.com/en/0.3.0/) we follow suggests only using:

* `Added`
* `Changed`
* `Deprecated`
* `Removed`
* `Fixed`
* `Security`

You should use `Changed` here.